README.md

```diff
@@ -1,2 +1,44 @@
-# js-libp2p-websocket-star-rendezvous
-The rendezvous service for libp2p-websocket-star enabled nodes meet and talk with each other
+# libp2p-websocket-star-rendezvous
+
+[![](https://img.shields.io/badge/made%20by-mkg20001-blue.svg?style=flat-square)](http://ipn.io)
+[![Travis](https://travis-ci.org/libp2p/js-libp2p-websocket-star-rendezvous.svg?style=flat-square)](https://travis-ci.org/libp2p/js-libp2p-websocket-star-rendezvous)
+[![Circle](https://circleci.com/gh/libp2p/js-libp2p-websocket-star-rendezvous.svg?style=svg)](https://circleci.com/gh/libp2p/js-libp2p-websocket-star-rendezvous)
+[![Coverage](https://coveralls.io/repos/github/libp2p/js-libp2p-websocket-star-rendezvous/badge.svg?branch=master)](https://coveralls.io/github/libp2p/js-libp2p-websocket-star-rendezvous?branch=master)
+[![david-dm](https://david-dm.org/libp2p/js-libp2p-websocket-star-rendezvous.svg?style=flat-square)](https://david-dm.org/libp2p/js-libp2p-websocket-star-rendezvous)
+
+> The rendezvous service for [libp2p-webrtc-star](https://github.com/libp2p/js-libp2p-websocket-star).
+
+## Descriptions
+
+Nodes using `libp2p-websocket-star` will connect to a known point in the network, a rendezvous point where they can learn about other nodes (Discovery) and route their messages to other nodes (2 hop message routing, also known as relay).
+
+## Usage
+
+`libp2p-websocket-star` comes with batteries included, that means that you can also spawn your own rendezvous point right from this module. To do that, first insall the module globally in your machine with:
+
+```bash
+> npm install --global libp2p-websocket-star
+```
+
+This will install a `websockets-star` CLI tool. Now you can spawn the server with:
+
+```bash
+> websockets-star --port=9090 --host=127.0.0.1
```

Comment:
Why is there an s? It's still named websocket-star

Reply:
It's a typo. I fixed it a bunch of times myself and still missed this one, my bad!